designer/app.py

```diff
@@ -922,15 +912,15 @@ def _fbrowser_load(self, instance):
             return
 
         file_path = instance.selection[0]
-        file_extension = instance.selection[0].split('.')
+        file_name, file_extension = os.path.splitext(instance.selection[0])
 
         error = None
         try:
-            if file_extension[1] == 'py':
+            if file_extension == '.py':
```

Comment:
@manthansharma can you update this line to support opening a project with kv, buidozer.spec and py?

something like:

`if file_extension in ('.py', '.kv') or (file_name.endswith('buildozer') and file_extension == 'spec')`